src/vs/workbench/contrib/terminal/browser/terminalMenus.ts

```diff
@@ -282,6 +316,36 @@ export function setupTerminalMenus(): void {
 					},
 					group: ContextMenuGroup.Config
 				}
+			},
+			{
+				id: MenuId.TerminalEditorInstanceContext,
+				item: {
+					command: {
+						id: TerminalCommandId.RevealInFinder,
+						title: localize('workbench.action.terminal.revealInFinder', "Reveal Current Working Directory in Finder"),
+					},
+					group: ContextMenuGroup.Navigation,
+				}
+			},
+			{
+				id: MenuId.TerminalEditorInstanceContext,
+				item: {
+					command: {
+						id: TerminalCommandId.RevealInExplorer,
+						title: localize('workbench.action.terminal.revealInExplorer', "Reveal Current Working Directory in Explorer"),
+					},
+					group: ContextMenuGroup.Navigation,
```

Comment:
Similar to editor tabs we want it to show lower in the list. How about below copy/paste and above clear (start of destructive actions)

![image](https://github.com/user-attachments/assets/6c750265-9fa2-41ce-aa39-99c1a706b23e)